src/components/SingleReview.jsx

```diff
@@ -0,0 +1,45 @@
+import { useParams } from "react-router-dom";
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+
+const SingleReview = () => {
+console.log(useParams());
+
+
+const { review_id } = useParams()
+const [singleReview, setSingleReview] = useState([]);
+
+
+useEffect(()=> {
+    axios
+    .get(`https://nialls-games-reviews.herokuapp.com/api/reviews/${review_id}`)
+    .then((response) => {
+        console.log(response);
+        setSingleReview(response.data.review, '<----')
+    //   setSingleReview(response.data);
+    
+    })
+  }, []);
+
+  return <>
+    <div>
+        <img id='singleReviewImage' src={singleReview.review_img_url}></img>
+    </div>
+        <ul className='singleReviewList'>
```

Comment:
< ul > elements are unordered lists containing < li > tags, not much point using this if you don't have and < li > elements inside. You could use a bigger variety of elements so they don't all look the same, for example < h3 > elements for your title.